cluster/ubuntu/util.sh

```diff
@@ -305,8 +308,8 @@ function kube-up() {
 
   # ensure the binaries are well prepared
   if [ ! -f "ubuntu/binaries/master/kube-apiserver" ]; then
-    echo "No local binaries for kube-up, downloading... "
-    "${KUBE_ROOT}/cluster/ubuntu/build.sh"
+    echo -n "No local binaries for kube-up, downloading..."
+    ${KUBE_ROOT}/cluster/ubuntu/download-release.sh
```

Comment:
From shellcheck.net: SC2086 (https://github.com/koalaman/shellcheck/wiki/SC2086) Double quote to prevent globbing and word splitting.